catalog/plugin/catalog-plugin-metacardbackup-filestorage/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/storage/filestorage/MetacardBackupFileStorage.java

```diff
@@ -0,0 +1,213 @@
+import org.codice.ddf.catalog.plugin.metacard.backup.storage.api.MetacardBackupStorageProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MetacardBackupFileStorage implements MetacardBackupStorageProvider {
+
+    private static final String DESCRIBABLE_PROPERTIES_FILE = "/describable.properties";
+
+    private static final String DESCRIPTION = "description";
+
+    private static final String ORGANIZATION = "organization";
+
+    private static final String VERSION = "version";
+
+    private static final String TITLE = "name";
+
+    private static final String ID = "id";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MetacardBackupFileStorage.class);
+
+    private static final String OUTPUT_DIRECTORY_PROPERTY = "outputDirectory";
+
+    private String id;
+
+    private String outputDirectory;
+
+    private static Properties describableProperties = new Properties();
+
+    static {
+        try (InputStream properties = MetacardBackupFileStorage.class.getResourceAsStream(
+                DESCRIBABLE_PROPERTIES_FILE)) {
+            describableProperties.load(properties);
+        } catch (IOException e) {
+            LOGGER.info(e.getMessage(), e);
+        }
+    }
+
+    public MetacardBackupFileStorage() {
+
+    }
+
+    @Override
+    public String getId() {
+        return id;
+    }
+
+    @Override
+    public String getVersion() {
+        return (String) describableProperties.get(VERSION);
+    }
+
+    @Override
+    public String getTitle() {
+        return (String) describableProperties.get(TITLE);
+    }
+
+    @Override
+    public String getDescription() {
+        return (String) describableProperties.get(DESCRIPTION);
+    }
+
+    @Override
+    public String getOrganization() {
+        return (String) describableProperties.get(ORGANIZATION);
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public void deleteData(String id) throws IOException, MetacardBackupException {
+        if (StringUtils.isEmpty(outputDirectory)) {
+            throw new MetacardBackupException(
+                    "Unable to delete stored data; no output directory specified.");
+        }
+
+        deleteBackupIfPresent(id);
+    }
+
+    public void storeData(String id, byte[] data) throws IOException, MetacardBackupException {
+        if (StringUtils.isEmpty(outputDirectory)) {
+            throw new MetacardBackupException("Unable to store data; no output directory specified.");
+        }
+
+        if (data == null) {
+            throw new MetacardBackupException("No data to store");
+        }
+
+        Path metacardPath = getMetacardDirectory(id);
+        if (metacardPath == null) {
+            LOGGER.debug(String.format("Unable to create metacard path directory for %s", id));
+            throw new MetacardBackupException("Unable to create metacard backup directory");
```

Comment:
Might be worthwhile to use the same string that you use in the debug statement here (includes id).  Otherwise you could drop `String.format` in the above line